tests/unit/transforms/attachment-test.js

```diff
@@ -0,0 +1,62 @@
+import { moduleFor, test } from 'ember-qunit';
+
+import Ember from 'ember';
+
+let testSerializedData = {
+  'test.txt': {
+    content_type: 'text/plain',
+    data: 'hello world!'
+  },
+  'stub.json': {
+    stub: true,
+    content_type: 'application/json'
+  }
+};
+
+let testDeserializedData = [
+  Ember.Object.create({
+    name: 'test.txt',
+    content_type: 'text/plain',
+    data: 'hello world!'
+  }),
+  Ember.Object.create({
+    name: 'stub.json',
+    content_type: 'application/json',
+    stub: true
```

Comment:
Don't know about PouchDB, but at least CouchDB seems to not care about their presence when writing the doc. A stub is a stub.

I guess it might throw a 409 if `digest`s don't match, hmm...